src/cityreader/cityreader.py

```diff
@@ -14,14 +21,26 @@
 #
 # Note that the first line of the CSV is header that describes the fields--this
 # should not be loaded into a City object.
+import csv
 cities = []
 
 def cityreader(cities=[]):
-  # TODO Implement the functionality to read from the 'cities.csv' file
-  # For each city record, create a new City instance and add it to the 
-  # `cities` list
-    
-    return cities
+
+  with open('cities.csv', newline='') as csvfile:
+    city_reader = csv.reader(csvfile, delimiter=',')
+    count = 0
+    for row in city_reader:
```

Comment:
Nice job. Another thing you could do is next(reader) to advance the reader to the first row after the header row.

Reply:
Thanks! That's a good point (to ignore the header row)!. I like your code reviews. I find them thorough and helpful. :-)